### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by hashknot**
Multiple backwards incompatible changes were merged recently. This PR updates the library documentation in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,46 +5,97 @@
...
+// Prints out:
+// http://demos-1.imgix.net/bridge.png?h=100&w=100
+// http://demos-2.imgix.net/flower.png?h=100&w=100
 ```
-    
-The UrlBuilder type also offers up a series of Properties:
 
-#### SignKey
-This is the private key for signing requests, as specified in your imgIX source.
+By default, shards are calculated using a checksum so that the image path always resolves to the same domain. This improves caching in the browser. However, you can supply a different strategy that cycles through domains instead. For example:
```
Comment (jayeb) [inline README.md]: Let's remove the mention of "cycle" and custom sharding strategies and just explain how Crc32 works here.

File: README.md [hunk 2]
```diff
@@ -5,46 +5,97 @@
...
+### Version 1.x
+
+v1.x of the library can be installed via NuGet:
+
+    Install-Package Imgix-CSharp
+
+
+## Basic Usage
+
+The library can be invoked by newing up UrlBuilder with your imgix domain. The URL builder can be reused to create URLs for any images on the domains it is provided.
```
[the remark this answers is not in the document]
Comment (hashknot) [reply README.md]: I used the previous text as is here. I will re-word it.

File: README.md (hunk 1 above)
Comment (hashknot) [reply README.md]: I didn't quite follow. Are you suggesting we don't mention anything about cyclic domain sharding or elaborate more on the CRC sharding or both?

Comment (hashknot) [reply README.md]: Got it.

**Pull request merged**